src/tool/subcommands/archive_cmd.rs

```diff
@@ -58,6 +58,27 @@ use std::sync::Arc;
 use tokio::io::{AsyncWriteExt, BufWriter};
 use tracing::info;
 
+#[derive(Debug, Clone, ValueEnum, Default)]
+pub enum ExportMode {
+    /// Export all types of snapshots.
+    #[default]
```

Comment:
We should remove the `#[default]` from here since we already have the `default_value_t = ExportMode::All` in the  flag and we will always get this value from the flag.